test/aws/scaleup.yml

```diff
@@ -32,6 +32,11 @@
       owner: root
       group: root
       mode: 0644
+  - name: Initialize openshift repos
+    import_role:
+      name: openshift_repos
```

Comment:
We are not running anything in openshift_repos to my knowledge.
```
PLAY [Setup yum repositories for all hosts] **************************************************************************************************
skipping: no hosts matched
```

Reply:
Can we defer this to BYOH-224? I'd like to get this code in so we don't have to rebase a lot next week.

Reply:
Fine with me.